src/scale.c

```diff
@@ -2,11 +2,28 @@
 
 
 cal_data_t blank_cal_data = {
+    // p_min and p_max
     0,
     16383,
+    // i_min and i_max
     0,
     16383,
-    { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 },
-    { 16383, 16383, 16383, 16383, 16383, 16383, 16383, 16383, 16383, 16383,
-      16383, 16383, 16383, 16383, 16383, 16383 }
+    // f_min
+    {
+        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+    },
```

Comment:
I believe these default f_min and f_max tables should have been expanded in https://github.com/monome/teletype/pull/288

Reply:
Or maybe at this point it would be less wasteful to initialize new `cal_data_t`s with code, and save all this program memory.

Reply:
(done)